addon/mixins/responds-to-resize.js

```diff
@@ -15,9 +15,10 @@ export default Ember.Mixin.create(
   didInsertElement: function () {
     this._super(...arguments);
 
-    this.resizeHandler = this.debounce(() => {
-      this.trigger('resize');
-      this.resize();
+    const self = this;
```

Comment:
Same reason as above, i.e. again we need access to `arguments` so we can't use an arrow function for the handler.